src/autosuggest/autosuggest-option.tsx

```diff
@@ -46,17 +43,14 @@ const AutosuggestOption = (
   ref: React.Ref<HTMLDivElement>
 ) => {
   const baseProps = getBaseProps(rest);
-  const i18n = useInternalI18n('autosuggest');
```

Comment:
Removed `useContext` call from every autosuggest option. Maybe performance tests will appreciate this effort